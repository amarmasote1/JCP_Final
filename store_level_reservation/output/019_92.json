{
    "warnings": null,
    "message": "Success",
    "nodeType": null,
    "body": {
        "dirty": false,
        "inventoryTimestamp": 1583216614000,
        "shipSafetyFactorSource": 0,
        "bopusSafetyFactor": 10,
        "shipSafetyFactor": 10,
        "onhandSupply": 100,
        "reservedDemand": 0,
        "pickSafetyFactorSource": 0,
        "unconfirmedDemand": 0
    },
    "key": "JCP|0652|16345678019|EACH",
    "status": 200
}