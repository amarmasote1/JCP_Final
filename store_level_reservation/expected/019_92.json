{
	"status": 200,
	"message": "Success",
	"key": "JCP|0652|16345678019|EACH",
	"body": {
		
		"shipSafetyFactor": 10,
		"bopusSafetyFactor": 10,
		"shipSafetyFactorSource": 0,
		"pickSafetyFactorSource": 0,

		
		
		
		"reservedDemand": 0,
		"unconfirmedDemand": 0,
		
		"dirty": false
	},
	"warnings": null
}